runtime/src/main/java/org/corfudb/runtime/view/stream/BackpointerStreamView.java

```diff
@@ -193,6 +197,29 @@ protected ILogData read(final long address) {
             throw te;
         }
     }
+    final long HOLE_FILL_THRESHOLD = 6000;
+
+    protected ILogData readData(final long address, long startTime) {
```

Comment:
it needs a comment can be made private method.

private ILogData read(final long address, long readStartTime)

Reply:
Comment:
Reads data  from an address in the address space. It will give the writer a chance to complete based on the time when the reads of which this individual read is a step started. If the reads have been going on for longer than the grace period given for a writer to complete a write, the subsequent individual read calls will immediately fill the hole on absence of data at the given address.